### PR TITLE
Manage users view

### LINKED ISSUE #2330: Create users list view
Similar to TM3, Tasking Manager 4 needs a users view to list information about users from database. The view needs:
- Filter user by role.
- Filter user by mapping level.
- search by username
- Show total of users found matching any of the filters.
- Create pagination.

### PR DESCRIPTION
**Pull request by JorgeMartinezG**
Closes #2330

### CODE REVIEW

File: frontend/src/components/userList.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+                    className="blue-grey mr2 ml3 link"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                    href={`/users/${user.username}`}
+                  >
+                    {user.username}
+                  </a>
+                </div>
+                <div className="w-30 fl">
+                  <span className="ttu dib-ns dn">
```
Comment (willemarcel) [inline frontend/src/components/userList.js]: I prefer without the uppercase

File: frontend/src/components/userList.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                    <FormattedMessage {...editMessages[`mapperLevel${user.mappingLevel}`]} />
+                  </span>
+                </div>
+                <div className="w-30 fl">
+                  <span className="ttu dib-ns dn">
```
Comment (willemarcel) [inline frontend/src/components/userList.js]: I prefer without the uppercase

File: frontend/src/views/users.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+export const UsersList = () => {
+  const [filters, setFilters] = useState({ level: 'ALL', role: 'ALL', username: '', page: 1 });
+
+  return (
+    <div>
+      <h3 className="barlow-condensed f2 ma0 pv3 dib v-mid ttu pl2 pl0-l">
+        <FormattedMessage {...messages.userList} />
+      </h3>
+      <div className="w-80 bg-white cf pa3 mb4">
+        <SearchNav filters={filters} setFilters={setFilters} />
```
Comment (willemarcel) [inline frontend/src/views/users.js]: Let's mode the SearchNav to above the users list to follow the same interface pattern:

image

image

image

**Comment by JorgeMartinezG** [comment]
@willemarcel branch updated with changes:
image

**Pull request merged**